<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>363</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>516</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-2</y>
     <width>221</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/images/register.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>301</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="login_welcome">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>欢迎登录QQ</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_common">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>每一天，乐在沟通。</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_username">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 134, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_error">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/images/error.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="login_attention">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名或密码错误</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="register_2">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLabel" name="register_welcome">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>欢迎注册QQ</string>
     </property>
    </widget>
    <widget class="QLabel" name="register_common">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>每一天，乐在沟通。</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_username">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 134, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_password_again">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>重复密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="register_error">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/images/error.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="register_attention">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名或密码不能为空</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
